client/src/Components/Navbar/index.css

```diff
@@ -0,0 +1,66 @@
+.FloatingButton {
```

Comment:
There's a lot of css that can be converted into tailwind here but while you're experimenting using normal css is fine.